### DISCUSSION TITLE
Project maintainers

### DISCUSSION THREAD

**Discussion by mywave82** [adplug/adplug#166] [General]
Hi

I am asking what the status of the maintainers are, and if the group perhaps should be enlarged. There are currently outstanding PR and bug requests that are simple bugfixes that pass by for some time now. I understand that changes that impact the design of the project or require a lot of work does not happen, but I hope that bugfixes and similiar could. And that there always are maintainers available iwithin some few weeks time, so that the project does not enter a full stale.

Best regards

**Comment by Malvineous** [comment]
The list of maintainers is at https://github.com/orgs/adplug/people

I've invited you to become an additional maintainer so you should have access to merge all the PRs - my apologies, I just haven't had the time lately to catch up on them all.  Hopefully you can review and merge the CMake ones too.

**Comment by mywave82** [comment]
I am honored, looking forward to help
